portfolio/src/main/webapp/index.html

```diff
@@ -72,20 +70,30 @@ <h1 class="header">About Me</h1>
       working as a Google STEP intern this summer.</p>
     <p> I love technology and creating awesome stuff!</p>
     <h2> Where I'm From</h2>
-    <p>I'm originally from Imo State, Nigeria. I currently reside in St. Paul, Minnesota</p>
-    <h2> More About Me</h2>
-    <h3>What are your favorite hobbies?</h3>
-    <p>In my free time, I enjoy reading, exercising, programming, and playing video games.</p>
-    <h3>What's your dream job?</h3>
+    <p>I currently reside in St. Paul, Minnesota</p>
+    <h2>My hobbies</h2>
+    <p>In my free time, I enjoy reading, drawing, programming, working out, and gaming.</p>
+    <h2>My your dream job?</h2>
     <p>My dream job is one where I create amazing technology that can have a positive 
       impact on the lives of many people!</p>
+    <h2>Fun facts!</h2>
+    <ul>
+        <li>I grew up in Nigeria and moved to the United States with family when I was 10. 
+            It was during this time that my first name was changed to Godsfavour 
+            (it was previously my middle name)</li>
+        <li>I love playing sports. I've been playing soccer my whole life, and I found track and field in high school.
+            One of my goals during college is to make the varsity track and field team for sprinting.</li>
+        <li>I'm the eldest of four in my family</li>
+        <li>My favorite genre of movies/tv shows is sci-fi. I'm currently watching the Stargate Atlantis, the third and last (so far) show of the Stargate series</li>
+    </ul>
   </div>
   
   <!-- Comments Section -->
   <a id="comments"></a>
   <div class="comments-section">
     <h1 class="header">Comments</h1>
     <div class="comments-container">
+        <h3><i>Leave a comment</i></h3>
```

Comment:
nit:  I would prefer css over `<i>`.  I think it makes it more consistent and easier to maintain.